simpleclient_logback/src/main/java/io/prometheus/client/logback/InstrumentedAppender.java

```diff
@@ -0,0 +1,84 @@
+package io.prometheus.client.logback;
+
+import ch.qos.logback.classic.Level;
+import ch.qos.logback.classic.spi.ILoggingEvent;
+import ch.qos.logback.core.Appender;
+import ch.qos.logback.core.UnsynchronizedAppenderBase;
+import io.prometheus.client.CollectorRegistry;
+import io.prometheus.client.Counter;
+
+public class InstrumentedAppender extends UnsynchronizedAppenderBase<ILoggingEvent> {
+
+  public static final String COUNTER_NAME = "logback_appender_total";
```

Comment:
This is only used in one place, put in the value rather than requiring someone reading this to jump around the code.